driver/docker/docker.go

```diff
@@ -218,25 +221,71 @@ func (d *Driver) exec(op *driver.Operation) error {
 		}
 	}()
 
-	statusc, errc := cli.Client().ContainerWait(ctx, resp.ID, container.WaitConditionRemoved)
+	statusc, errc := cli.Client().ContainerWait(ctx, resp.ID, container.WaitConditionNotRunning)
 	if err = cli.Client().ContainerStart(ctx, resp.ID, types.ContainerStartOptions{}); err != nil {
-		return fmt.Errorf("cannot start container: %v", err)
+		return driver.OperationResult{}, fmt.Errorf("cannot start container: %v", err)
 	}
 	select {
 	case err := <-errc:
 		if err != nil {
-			return fmt.Errorf("error in container: %v", err)
+			opResult, _ := d.fetchOutputs(ctx, resp.ID)
+			return opResult, fmt.Errorf("error in container: %v", err)
 		}
 	case s := <-statusc:
 		if s.StatusCode == 0 {
-			return nil
+			return d.fetchOutputs(ctx, resp.ID)
 		}
 		if s.Error != nil {
-			return fmt.Errorf("container exit code: %d, message: %v", s.StatusCode, s.Error.Message)
+			opResult, _ := d.fetchOutputs(ctx, resp.ID)
```

Comment:
This error is dropped, so the user won't be able to understand why she can't get her logs if the operation fails (which can occur as the container itself failed). I guess we should somehow merge the 2 errors 🤔

Reply:
Makes sense, I merge those errors.

Reply:
@youreddy I see this comment marked as resolved but couldn't see the associated commit 🤔

Reply:
@silvin-lubecki sorry about that, @astrieanna might have resolved it before I pushed it up. You should be able to see it now!